docs/FileFormats.md

```diff
@@ -71,6 +71,7 @@ The `"predictions"` list contains objects for each prediction, and each object c
     - `"point`: A numeric point prediction with a single `value` key.
     - `"sample"`: Numeric samples represented as a table with one column that is found in the `sample` key.
     - `"quantile"`: A quantile distribution with two paired columns: `quantile` and `value`.
+    - `"mean"`, `"median"`, and `"mode"`: Similar to `"point"` but with clear semantics.
```

Comment:
I suggest making this line stand on its own, if nothing else to ready ourselves for a removal of the "point" row that might be removed at a later point. So maybe the text after the full colon could read:
"A numeric prediction with a single `value` key, indicating the summary statistic indicated by the name, e.g. the mean."